src/Sulu/Bundle/WebsiteBundle/Sitemap/SitemapUrl.php

```diff
@@ -68,9 +73,10 @@ class SitemapUrl
      * @param string $changefreq
      * @param float $priority
      */
-    public function __construct($loc, \DateTime $lastmod = null, $changefreq = null, $priority = null)
+    public function __construct($loc, $locale, \DateTime $lastmod = null, $changefreq = null, $priority = null)
```

Comment:
@chirimoya @wachterjohannes That's the parameter I've been talking about. It should be mandatory, because based on it the URL in the twig template is generated. I could move it to the very end, which would also be a BC break. Except we don't make this parameter mandatory, but I think that's not the way to go.